src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { Card, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { BrowserRouter as Router, Route, Redirect, Link } from "react-router-dom";
+import axios from 'axios';
+
+export function ProfileView(props) {
+    console.log('props: ', props);
+
+    const [userData, setUserData] = useState({});
+
+    console.log('fav movies');
+
+    function getUserData(token) {
+        console.log('get user data');
+        axios.get('https://morning-badlands-52426.herokuapp.com/users/' + props.user, {
+            headers: { Authorization: `Bearer ${token}` }
+        })
+            .then(response => {
+                console.log('user', response.data);
+                // Assign the result to the state
+                setUserData(response.data);
+            })
+            .catch(function (error) {
+                console.log(error);
+            });
+    }
+
+    useEffect(() => {
+        console.log('use effect');
+        getUserData(localStorage.getItem('token'))
```

Comment:
♻️ You dont have to pass the token in here. It is much cleaner to grab the token from the getUserData function above.
```js
function getUserData(token) {
        console.log('get user data');
        const token = localStorage.getItem('token');
        axios.get('https://morning-badlands-52426.herokuapp.com/users/' + props.user, {
            headers: { Authorization: `Bearer ${token}` }
        })
```

Reply:
updated: function recycled token obtained from getUserData function